### PR TITLE
Plugged in the new translations

### PR DESCRIPTION
**Pull request by trevorjonesd2l**

### CODE REVIEW

File: lang/de.html [hunk 1]
```diff
@@ -55,9 +55,13 @@
 			'filtering.noRoles': 'Sie haben keine Rollenfilter ausgewählt.',
 			'filtering.roles': 'Rollen',
 			'overlay.courseStarting': 'Kurs beginnt',
-			'overlay.courseStartingOn': 'Kurs beginnt am {date} {time}',
+			'overlay.courseStartingOn': 'Kurs beginnt am {date} um {time}',
```
Comment (dlockhart) [inline lang/de.html]: Seems like they put an "at" between the date and time here? "um" in German -> "around" in English.

File: lang/fr.html [hunk 2]
```diff
@@ -55,9 +55,13 @@
 			'filtering.noRoles': 'Vous n\'avez aucun filtre de rôles.',
 			'filtering.roles': 'Rôles',
 			'overlay.courseStarting': 'Le cours commence',
-			'overlay.courseStartingOn': 'Le cours commence le {date} {time}',
+			'overlay.courseStartingOn': 'Le cours commence le {date} à {time}',
```
Comment (dlockhart) [inline lang/fr.html]: French also put an "at" in here.

File: lang/de.html (hunk 1 above)
Comment (trevorjonesd2l) [reply lang/de.html]: Yeah, maybe we should just remove this?

Comment (dlockhart) [reply lang/de.html]: I think so, yes, and same for the French version.

Comment (ryantmer) [reply lang/de.html]: I don't speak German, but can confirm that removing it in French isn't problematic. It's the same level of weird as removing it in English, i.e. a little non-conversational but that's about it.

**Pull request merged**